### DISCUSSION TITLE
Full Example

### DISCUSSION THREAD

**Discussion by mariohmol** [jellydn/next-swagger-doc#107] [Q&A]
Hi,

would be awesome if we have a full example making the docs, maybe something simulating the swagger example? https://petstore.swagger.io/v2/swagger.json

One question is, how to create the docs for generic data like:
* tags
* schemas

I´ve trying to make it here like this way:

```
/**
 * @swagger
 * definitions:
 *   Faker:
 *     type: 'object'
 *     properties:
 *       email:
 *         type: string 
 * tags:
 *   -
 *     name: 'faker'
 *     description: Cria valores fictícios para testes
 *     externalDocs:
 *       description: Cria valores fictícios para testes
 * 
 * /api/faker/cpf:
 *   get:
 *     tags: ['faker']
 *     description: Returns the hello world
 *     responses:
 *       200:
 *         description: hello world
 *         schema:
 *           $ref: '#/definitions/Faker'
 */
```

**Comment by jellydn** [comment]
Hi @mariohmol I have implemented an example but that's a simple one. 

```
mkdir test
npx degit jellydn/next-swagger-doc/example
yarn && yarn build && yarn start
```

**Reply by jellydn** [reply]
I'm quite busy not but will try to create a full example later.

**Comment by HunteRoi** [comment]
Hey guys, I have started using NextJS and this project as well yesterday. I'm quite new to developing a backend in JavaScript like this and especially using Swagger.

Is it possible to have an explanation about how to create tags so that I can assign tags to my routes? I currently have something like this:
![image](https://user-images.githubusercontent.com/32441291/146650065-973c52cc-cd5c-437a-a4f4-bee5517e1c4c.png)

and I would like to have the related `supervisors` routes under the "Supervisor" tag; `levels` routes under `Level` tag.

Kind regards!
